salt/top.sls

```diff
@@ -70,6 +70,10 @@ base:
     - match: grain
     - pulsar
     - pulsar.tests
+  'roles:dagster':
+    - match: grain
+    - dagster
+    - nginx
```

Comment:
Should the `data` environment be added to [here](https://github.com/mitodl/salt-ops/blob/main/salt/top.sls#L10) so that consul get installed with prod configs?

Reply:
Eventually we will want to do that, but for the scope of this PR I'm mainly focusing on just getting the AMI to build. There will likely be other changes needed for the final deployment which we can bundle into a separate PR.